hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -26,20 +29,40 @@ class HassioIngressView extends LitElement {
 
   @internalProperty() private _addon?: HassioAddonDetails;
 
+  @property({ type: Boolean })
+  public narrow = false;
+
   protected render(): TemplateResult {
     if (!this._addon) {
       return html` <hass-loading-screen></hass-loading-screen> `;
     }
+    const inConfigPanel = location.pathname.startsWith("/hassio/ingress");
+
+    if (
+      !inConfigPanel &&
+      !this.narrow &&
+      this.hass.dockedSidebar !== "always_hidden"
+    ) {
+      return html` <iframe src=${this._addon.ingress_url!}></iframe> `;
+    }
 
     return html`
-      <hass-subpage .header=${this._addon.name} hassio>
-        <iframe src=${this._addon.ingress_url}></iframe>
+      <hass-subpage
```

Comment:
I think that we shouldn't use a hass-subpage. Just a div should be enough. 

I still think that we should be able to use the look of iOS. Maybe we can make it just 40px high, so it is still valid touch target, but it doesn't take up too much height. And just make the iFrame height(100% - 40px).